Game/Player.pde

```diff
@@ -3,8 +3,10 @@ class Player {
   public static final int MOVE_TIME = 30;
 
   Board board;
-
+  
+//-----------------------------------------------------PLEASE MOVE IT TO ANOTHER CLASS-------------------------------
   Inventory inventory;
+//-------------------------------------------------------------------------------------------------------------------

```

Comment:
You don't think Inventory should be a part of the Player class? I thought it would make sense in the OOP way of thinking.

What do you think about moving it to the Game class?